container-chains/templates/simple/node/src/cli.rs

```diff
@@ -181,3 +186,16 @@ impl CliConfiguration for BuildSpecCmd {
         Some(&self.base.node_key_params)
     }
 }
+
+#[derive(Debug, clap::Parser)]
+#[group(skip)]
+pub struct RpcProviderSubcommand {
+    /// Endpoints to connect to orchestrator nodes, avoiding to start a local orchestrator node.
+    /// If this list is empty, a local embeded orchestrator node is started.
+    #[arg(long)]
+    pub orchestrator_endpoints: Vec<Url>,
+
+    /// Account associated with the node, whose assignements will be followed to provide RPC services.
+    #[arg(long)]
+    pub assignement_account: dp_core::AccountId,
```

Comment:
we probably dont need this account as we can take it from the node key, but let's leave it for now yes